math/src/omnipool/math.rs

```diff
@@ -110,19 +103,26 @@ pub fn calculate_sell_hub_state_changes(
 
 	let amount_out = to_balance!(delta_reserve_out_hp).ok()?;
 	let delta_reserve_out = amount_without_fee(amount_out, asset_fee)?;
-	let asset_fee = amount_out.saturating_sub(delta_reserve_out);
-
-	let delta_imbalance = calculate_imbalance_in_hub_swap(total_hub_reserve, hub_asset_amount, imbalance)?;
+	let asset_fee_amount = amount_out.saturating_sub(delta_reserve_out);
+
+	// mint amount to account for asset fee that stays in the pool
```

Comment:
this comment is still misleading for me, are we minting here, or just calculating how much we wanna mint?

Reply:
it is not a verb here. it is mint amount. i change